pkg/katautils/config.go

```diff
@@ -602,6 +614,91 @@ func newQemuHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 	}, nil
 }
 
+func newAcrnHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
+	hypervisor, err := h.path()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	hypervisorctl, err := h.ctlpath()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	kernel, err := h.kernel()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	image, err := h.image()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	if image == "" {
+		return vc.HypervisorConfig{},
+			errors.New("image must be defined in the configuration file")
+	}
+
+	firmware, err := h.firmware()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	machineAccelerators := h.machineAccelerators()
+	kernelParams := h.kernelParams()
+	machineType := h.machineType()
+
+	blockDriver, err := h.blockDeviceDriver()
+	if err != nil {
+		return vc.HypervisorConfig{}, err
+	}
+
+	useVSock := false
+	if h.useVSock() {
+		if utils.SupportsVsocks() {
+			kataUtilsLogger.Info("vsock supported")
+			useVSock = true
+		} else {
+			kataUtilsLogger.Warn("No vsock support, falling back to legacy serial port")
+		}
+	}
+
+	//TODO: Need to trim this config specific to ACRN
```

Comment:
raise an issue and mention it here

Reply:
Trimmed HV config passed to ACRN and removed TODO.